src/docs/product/relay/options.mdx

```diff
@@ -6,34 +6,42 @@ redirect_from:
-| relay.mode                 | String  | default value: `managed` possible values: `managed, static, proxy, capture` optional: `false` | Controls how Relay obtains the project configuration for events. For detailed explanation of these modes, see [Relay Modes](/product/relay/modes/).                                                                                                                                                                                                                                                                                             |
-| relay.upstream             | String  | default value: `https://sentry.io` optional: `false`                                          | Fully qualified URL of the upstream Relay or Sentry instance. <Alert level="warning" title="Important"> Relay doesn't check for cycles. Don't set this option to an endpoint that will cause events to be cycled back here. </Alert>                                                                                                                                                                                                            |
-| relay.host                 | String  | default value: `0.0.0.0  in Docker, otherwise  127.0.0.1` optional: `false`                   | The host to which Relay should bind (network interface).                                                                                                                                                                                                                                                                                                                                                                                        |
-| relay.port                 | Integer | default value: `3000` optional: `false`                                                       | The port to which Relay should bind for the unencrypted Relay HTTP server.                                                                                                                                                                                                                                                                                                                                                                      |
+| relay.mode                 | String  | default value: `managed` possible values: `managed, static, proxy, capture` optional: `false` | Controls how Relay obtains the project configuration for events. For detailed explanation of these modes, see [Relay Modes](/product/relay/modes/). `Environment variable: "RELAY_MODE."`                                                                                                                                                                                                                                                       |
```

Comment:
If we're creating a table, I think it'd be less confusing if the environment variable is another column like `key`, instead of a note in the description. See how we do with [Next.js docs](https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#use-configuration-files-for-source-map-upload), as an example.

Reply:
Ah yes, I tend to agree. I think the more we can split out the information the easier it will be to read <3

Reply:
this was my plan, but with 5 columns it will crash with the "ON THIS PAGE" index, is there a way to go around this? @lizokm

Reply:
@TBS1996 fixing the overlap of the "on this page" index is in our backlog to figure out, but we haven't fixed it yet, unfortunately. So I think we should just leave it as 4 columns for now and then we don't have to worry about the Size Limits table having one lonely entry :) 

In any case, I think this change will make the page much easier to read even if we don't have a separate column for environment variables!